Gems/ROS2/Code/Source/RobotImporter/SDFormat/ROS2SDFormatHooksUtils.cpp

```diff
@@ -98,4 +100,166 @@ namespace ROS2::SDFormat
                 "CreatePrefabFromUrdfOrSdf", "Setting transform failed: %s does not have transform interface\n", sdfSensor.Name().c_str());
         }
     }
+
+    namespace HooksUtils::PluginParser
+    {
+        // Inserts name (key) and value (val) of given parameter to map.
+        void ParseRegularContent(const sdf::Element& content, HooksUtils::PluginParams& remappings)
+        {
+            std::string contentName = content.GetName();
+            std::string contentValue = content.GetValue()->GetAsString();
+            if (contentName.empty() || contentValue.empty())
+            {
+                AZ_Warning("PluginParser", false, "Encountered empty parameter value while parsing URDF/SDF plugin.");
+                return;
+            }
+            AZStd::string key(contentName.c_str(), contentName.size());
+            AZStd::string val(contentValue.c_str(), contentValue.size());
+            remappings[key] = val;
+        }
+
+        // Parses parameters present in <ros> element, inserting them to the map.
```

Comment:
I believe that bracket are special characters for doxygen (and perhaps other documentation generators), default for unrecognized tags is to skip them. IIRC, the output in generated docs would be "Parses parameters present in  element, inserting them to the map".

Please double check and escape if necessary, or refer to the tags themselves without brackets. (Please check other occurrences of this issue).